packages/components/src/Dialog/Dialog.test.js

```diff
@@ -78,6 +84,10 @@ describe('Dialog', () => {
 		const wrapper = shallow(<Dialog {...subtitleProps}>{children}</Dialog>);
 		expect(wrapper.getElement()).toMatchSnapshot();
 	});
+	it('should render icon and subtitle', () => {
+		const wrapper = shallow(<Dialog {...iconAndSubtitleProps}>{children}</Dialog>);
+		expect(wrapper.getElement()).toMatchSnapshot();
```

Comment:
Instead of snapshot is it possible to simply check if `<Icon />` is present? 

Personally I feel like that has a higher chance to prevent regression than a snapshot, because nobody likes looking at snapshots 😄